apis/apps/v1beta3/emqxbroker_types.go

```diff
@@ -22,6 +22,10 @@ import (
 )
 
 type EmqxBrokerTemplate struct {
+	// Registry will used for EMQX owner image,
+	// like ${registry}/emqx/emqx:4.4.11 and ${registry}/emqx/emqx-operator-reloader,
```

Comment:
```suggestion
	// like ${registry}/emqx/emqx and ${registry}/emqx/emqx-operator-reloader,
```
My bad, maybe no tag better

Reply:
done